src/scripts/load.py

```diff
@@ -0,0 +1,75 @@
+"""
```

Comment:
How are you supposed to run this script? From the command line?

Reply:
Yes, it is run with the command "python manage.py runscript load" I can add this in the script or create a "how to" section on running the script?

Reply:
I think putting this sort of thing in the docstring and the howto would be good.

Reply:
You could put an example usage in the docstring:

https://google.github.io/styleguide/pyguide.html#382-modules

Reply:
I have made this change. Thank you!